presto-main/src/main/java/com/facebook/presto/operator/repartition/MapBlockEncodingBuffer.java

```diff
@@ -222,6 +222,8 @@ public void noMoreBatches()
             bufferAllocator.returnArray(offsets);
             offsets = null;
         }
+
+        columnarMap = null;
```

Comment:
What about the columnarArray and columnarRow in their specific buffers?

Reply:
Can you point which field you mean here?

Reply:
@pranjalssh Never mind, these are not persisted in their specific classes. Sorry for the confusion.